src/core/__tests__/ReactCompositeComponent-test.js

```diff
@@ -485,7 +485,49 @@ describe('ReactCompositeComponent', function() {
     );
   });
 
-  it('should pass context', function() {
+  it('should pass context to children when not owner', function() {
+    var childInstance = null;
```

Comment:
childInstance is unused -- grandchildInstance is fine I guess but you could also use React.findDOMNode on the returned component and check the element from that instead of what you did here.